app/build.gradle

```diff
@@ -102,7 +103,7 @@ dependencies {
     compile 'com.getpebble:pebblekit:3.0.0'
     compile 'io.reactivex:rxjava:1.0.0'
     compile 'com.github.lecho:hellocharts-android:v1.5.5'
-
+    compile 'com.google.android.gms:play-services-wearable:7.5.0'
```

Comment:
That's a pretty old version of play services.  Does it still not work on newer versions?